react/components/CreateOrganizationButton/CreateOrganizationButton.tsx

```diff
@@ -57,13 +59,14 @@ const CreateOrganizationButton = () => {
           icon={<IconPencil />}
           onClick={() => setOpen(true)}
         />
-        {!!data && (
+        <ImportInBulkTooltip visible={!!data}>
           <MenuItem
             label={formatMessage(messages.addBulk)}
             icon={<IconCloudArrowUp />}
             onClick={() => setUploadModalOpen(true)}
+            disabled={!data}
```

Comment:
I believe that Admin UI already handles this, but can you double check that the `onClick` handler is indeed ignored when `disabled=true` here? I've seen bad implementations where disabled items would still work when clicked, so I always double check these use cases.

Reply:
Right! I double checked, the admin-ui really handle this and the onClick is not emitted when the button is disabled